extensions/azurePublish/src/node/schema.ts

```diff
@@ -30,6 +30,31 @@ const schema: JSONSchema7 = {
       title:
         'Runtime identifier for hosting bot, default to win-x64, please refer to https://docs.microsoft.com/en-us/dotnet/core/rid-catalog',
     },
+    subscriptionId: {
+      type: 'string',
+      title: 'Id of your subscription'
+    },
+    resourceGroup: {
+      type: 'string',
+      title: 'Name of your resource group'
+    },
+    botName: {
+      type: 'string',
+      title: 'Name of your bot channel registration service'
+    },
+    abs: {
```

Comment:
QnA is missing from schema and there is no abs section now, all of the settings from abs are under root.

Reply:
qna and other resources added.